cnf-certification-test/networking/netcommons/netcommons.go

```diff
@@ -149,18 +149,20 @@ func FilterIPListByIPVersion(ipList []string, aIPVersion IPVersion) []string {
 	return filteredIPList
 }
 
-func FindRogueContainersDeclaringPorts(containers []*provider.Container, portsToTest map[int32]bool, portsOrigin string) (compliantObjects, nonCompliantObjects []*testhelper.ReportObject) {
+func findRogueContainersDeclaringPorts(containers []*provider.Container, portsToTest map[int32]bool, portsOrigin string, logger *log.Logger) (compliantObjects, nonCompliantObjects []*testhelper.ReportObject) {
 	for _, cut := range containers {
+		logger.Info("Testing Container %q", cut)
 		for _, port := range cut.Ports {
 			if portsToTest[port.ContainerPort] {
-				log.Debug("%s has declared a port (%d) that has been reserved", cut, port.ContainerPort)
+				logger.Error("%q declares %s reserved port %d (%s)", cut, portsOrigin, port.ContainerPort, port.Protocol)
```

Comment:
Should it be Debug?

Reply:
No. A container declaring a reserved port is an error.